### PR TITLE
https added

### PR DESCRIPTION
**Pull request by codecoder005**

### CODE REVIEW

File: src/main/resources/application-dev.yml [hunk 1]
```diff
@@ -1,4 +1,14 @@
 logging:
   file:
     name: ./logs/dev/application-dev.log
-    max-file-size: 100KB
\ No newline at end of file
+    max-file-size: 100KB
+
+server:
+  port: 8443
```
Comment (Raakhi130937) [inline src/main/resources/application-dev.yml:7]: should we always run the app on port 8443?

File: src/main/resources/static/error.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
```
Comment (Raakhi130937) [inline src/main/resources/static/error.html:1]: you have added the error.html file in the static folder. is that correct?

File: src/main/resources/static/index.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>hello-world-repo</title>
+</head>
+<body>
+  <div>
+    <div>
+        <h1>Hello World!</h1>
```
Comment (Raakhi130937) [inline src/main/resources/static/index.html:10]: can you change the welcome message?

**Review by Raakhi130937 — COMMENTED** [review]
look into below comments.

File: src/main/resources/application-dev.yml (hunk 1 above)
Comment (codecoder005) [reply src/main/resources/application-dev.yml:7]: yes

File: src/main/resources/static/error.html (hunk 2 above)
Comment (codecoder005) [reply src/main/resources/static/error.html:1]: yes

File: src/main/resources/static/index.html (hunk 3 above)
Comment (codecoder005) [reply src/main/resources/static/index.html:10]: no. that is fine.

File: src/main/resources/application-dev.yml (hunk 1 above)
Comment (Raakhi130937) [inline src/main/resources/application-dev.yml:7]: since you are saying 8443 as the port number. expose the same in Dockerfile.

**Review by Raakhi130937 — COMMENTED** [review]
expose same port in Dockerfile

File: src/main/resources/application-dev.yml (hunk 1 above)
Comment (codecoder005) [reply src/main/resources/application-dev.yml:7]: changed as requested. please check

**Review by Raakhi130937 — APPROVED** [review]
looks fine. approved.

**Pull request merged**